integration-tests/testutils/cli/cli_test.go

```diff
@@ -420,6 +420,20 @@ func (s *cliTestSuite) TestAddOptionsToSnappyCommand(c *check.C) {
 	}
 }
 
+func (s *cliTestSuite) TestAddOptionsDoesNotModifyOriginalCmds(c *check.C) {
```

Comment:
Nope, there was a problem with the way the coverage options were inserted and the original input `cmds` slice was modified. The problem has arisen after feeding a snap command to `wait.ForCommand`, when the same cmds slice is called more than once.

It's easy to remove the coverage options from the command we print, I can propose that in another branch. IMO we should wait to see how goes the upcoming verbosity flag branch.